services/core-api/app/api/mines/permits/permit_amendment/models/permit_amendment.py

```diff
@@ -13,10 +13,14 @@
 
 from . import permit_amendment_status_code, permit_amendment_type_code
 from app.api.utils.models_mixins import AuditMixin, Base
+from app.api.constants import *
 
 
 class PermitAmendment(AuditMixin, Base):
     __tablename__ = 'permit_amendment'
+    _edit_groups = [PERMIT_EDIT_GROUP, PERMIT_AMENDMENT_EDIT_GROUP, MINE_EDIT_GROUP]
```

Comment:
Same comment about the mine_edit_group and permits.

Reply:
changed